src/Microsoft.ML.AutoML/AutoMLExperiment/IPerformanceMonitor.cs

```diff
@@ -110,9 +108,18 @@ private void SampleCpuAndMemoryUsage()
                 // calculate Memory Usage in MB
                 var memoryUsage = process.PrivateMemorySize64 * 1.0 / (1024 * 1024);
                 _peakMemoryUsage = Math.Max(memoryUsage, _peakMemoryUsage ?? 0);
+
```

Comment:
Thanks for making this change by aggregating all performance metrics into one class

Reply:
No problem!